src/components/SliderBox/components/SliderCard/SliderCard.scss

```diff
@@ -0,0 +1,25 @@
+.slider-card {
+  display: flex;
+  border: 1px #90caf9 solid;
+}
+
+.slider-card-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.slider-card-content-container {
+  flex: 1 1 auto;
+}
+
+.slider-card-footer {
+  align-items: center;
+  align-self: flex-end;
+}
+
+.slider-card-read-more-link {
+  color: #212121;
```

Comment:
look though described approach https://github.com/kerrimov/the-ssspace/pull/4#discussion_r850394614

Reply:
fixed